tokio-trace-fmt/src/default.rs

```diff
@@ -10,44 +14,163 @@ use std::fmt::{self, Write};
 #[cfg(feature = "ansi")]
 use ansi_term::{Colour, Style};
 
-pub fn fmt_event<N>(ctx: &span::Context<N>, f: &mut Write, event: &Event) -> fmt::Result
+/// A type that can measure and format the current time.
+///
+/// This trait is used by `Standard` to include a timestamp with each `Event` when it is logged.
+///
+/// Notable default implementations of this trait are `SystemTime` and `()`. The former prints the
+/// current time as reported by `std::time::SystemTime`, and the latter does not print the current
+/// time at all. `FormatTime` is also automatically implemented for any function pointer with the
+/// appropriate signature.
```

Comment:
Nit/TIOLI: personally, I'd prefer to wrap these at the 80th column, but I'm fine with this wrapping.